app/views/admin/includes/_messages.html.erb

```diff
@@ -1,10 +1,57 @@
+<% if flash[ :notice ] %>
```

Comment:
Consider removing the duplication from this template.

Reply:
Like so? https://github.com/denny/ShinyCMS-ruby/commit/ed35c1d2957860f2a4537e6e4304cd23ea133c08

Reply:
Good start, there's a tad more possible https://github.com/denny/ShinyCMS-ruby/pull/78#discussion_r349530168